src/utils/erc20.ts

```diff
@@ -0,0 +1,25 @@
+import BN from 'bn.js';
+
+import { DEFAULT_NETWORK_CHAINID } from '@config';
+import { ERC20 } from '@services';
+import { ITxData, ITxFromAddress, ITxGasPrice, ITxObject, ITxToAddress, TAddress } from '@types';
+import { inputValueToHex } from '@utils'
+
+export const formatApproveTx = (
```

Comment:
Can we make it take an object with the params instead? It helps with readability when using it!